Server/src/usercoordinator.erl

```diff
@@ -66,24 +66,49 @@ loop(Socket, Diagrams) ->
 	{class_diagram, Diagram} ->
 	  Format_result = io_lib:format("~p", [Diagram]) ++ "~",
 		gen_tcp:send(Socket, Format_result),
-		loop(Socket, Diagrams)
-	  
+		loop(Socket, Diagrams);
+		
+	%Sends a deployment diagram client-side.
+	{deployment_diagram, DeploymentDiagram} ->
+      %Formats the message into a sendable format.
+	  io:format("Who dis"),
```

Comment:
More prints - that are helpfull when testing - but probably should be removed before merge.